app/server.js

```diff
@@ -17,13 +17,19 @@ Object.keys(config).map((configKey) => {
 // Routes
 const authProvider = new AuthProvider();
 const app = new App(authProvider).createApp();
-app.use('/', routes(authProvider));
+app.use(`/${config.baseUrl}`, routes(authProvider));
+if (config.baseUrl) {
```

Comment:
what is this bit for?

Reply:
redirects from / to /admin if there's a base url defined in config

Reply:
would it be better to redirect only if the env is `development` rather than if it is defined in config?

Reply:
We can't use the env variables because in the future we will be developing with ingress on